### ISSUE TITLE
Author lookup does not add authors to Item 

### ISSUE THREAD

**Issue by MarisArins** [DSpace/dspace-angular#867]
1. Start creating new Item
2. Enter authorh name/surname by hand, hit "+"  and author line appears on page
3. Go to Author lookup and pick any combinations of "local" authors
4. Selected author count is displayed at bottom, the list of authors can be seen in "Current selection" tab
5. Close the lookup - no new lines get added to author list
6. If you open the Lookup again, the "Current selection" displays correct list of authors previously selected, as well in the "Local authors" tab you still see the selections.

Expected : 
Selected authors would be added to the Item's list of authors

**Comment by tdonohue** [comment]
This sounds to me like a possible caching issue in the submission UI.  It _sounds_ like the authors are being added, but just don't appear on the submission form.  In any case, added to beta 5 for analysis/fixing

**Comment by tdonohue** [comment]
I'm able to reproduce this on the Demo site at https://dspace7-demo.atmire.com/  It looks like it has to do with Person Entities ("Local Authors" in the Lookup modal).  Whenever I select Local Authors & close the modal, they do **not** appear on the form after closing the modal.

If I open Chrome DevTools while selecting these authors, I see an error is logged in the console each time I select a checkbox.  That error is:
```
ERROR TypeError: Cannot read property 'uuid' of undefined
    at t.checkIfValidInput (filter-input-suggestions.component.ts:80)
    at Object.handleEvent (filter-input-suggestions.component.ngfactory.js.pre-build-optimizer.js:48)
    at Object.handleEvent (core.js.pre-build-optimizer.js:29239)
    at Object.handleEvent (core.js.pre-build-optimizer.js:29784)
    at qp (core.js.pre-build-optimizer.js:19859)
    at core.js.pre-build-optimizer.js:21044
    at t.i [as _next] (core.js.pre-build-optimizer.js:23747)
    at t.__tryOrUnsub (Subscriber.js.pre-build-optimizer.js:194)
    at t.next (Subscriber.js.pre-build-optimizer.js:132)
    at t._next (Subscriber.js.pre-build-optimizer.js:76)
``` 

As a sidenote, I also see that several *tabs* in the Lookup modal have no i18n keys, as the entire key is shown for `submission.sections.describe.relationship-lookup.search-tab.tab-title.pubmed` and `submission.sections.describe.relationship-lookup.search-tab.tab-title.arxiv`.  This likely can be easily fixed at the same time.

@artlowel : as promised in today's meeting, I'm assigning this to you (or your team) to estimate.  It sounds like a small bug to me.

**Comment by tdonohue** [comment]
This may be the same error as #897 as PR #898 looks like it's fixing the exact line mentioned in the above error message.

**Comment by artlowel** [comment]
The issue seems to be that the first collection you see when creating a new Item: [Cases](https://dspace7-demo.atmire.com/collections/2e971b93-d0c1-46a5-9425-0370882ad5fe) (and most other collections on the demo site, a lot have been added by users) didn't have an item template to set the relationship.type metadata field. Without it you can't create relationships. I've [added it for that collection](https://dspace7-demo.atmire.com/collections/2e971b93-d0c1-46a5-9425-0370882ad5fe/itemtemplate) and now it works for that collection.

The `ERROR TypeError: Cannot read property 'uuid' of undefined` error will indeed occur everywhere there's facets and will likely be solved by #898 I'll review it.

So the only thing left to do for _this_ PR is to add the i18n labels for the new external sources that have been added on the rest side, which can be done in **1h**.

We'll create a PR for that
